ci/citest/acceptance-test/tests/cmd_console_test.go

```diff
@@ -23,3 +23,14 @@ func TestCmdConsole_ShowOption(t *testing.T) {
 	RunCmdWithTimeoutAndReportFail(t, 10, 5, "openvdc", "destroy", instance_id)
 	WaitInstance(t, 5*time.Minute, instance_id, "TERMINATED", nil)
 }
+
+func TestCmdConsole_Authentication(t *testing.T) {
+	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc", `{"authentication_type":"none"}`)
+	instance_id := strings.TrimSpace(stdout.String())
+
+	WaitInstance(t, 5*time.Minute, instance_id, "RUNNING", []string{"QUEUED", "STARTING"})
+
+
+	RunCmdWithTimeoutAndReportFail(t, 10, 5, "openvdc", "destroy", instance_id)
+	WaitInstance(t, 5*time.Minute, instance_id, "TERMINATED", nil)
```

Comment:
How is this a test for console authentication? It just starts an instance and then terminates it.

Reply:
Added code to test the console. And change function name to TestCmdConsole_ShowOptionAuthenticationNone.